packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginActions.test.ts

```diff
@@ -231,9 +231,29 @@ describe('createPluginActions', () => {
         field: 'dummy',
         callbackId: TEST_CALLBACK_ID,
         filename,
+        fieldtype: 'asset',
+        name: '',
+        meta_data: {},
+        title: '',
+        copyright: '',
+        focus: '',
+        alt: '',
+        source: '',
+        is_private: false,
```

Comment:
Hey @eunjae-lee 👋 

I don't know how feasible it would be but would be possible to also have the `is_external_url` property added? 🤔 
May it not be possible or not make sense but just commenting once I saw this property exists in the image shared on the [issue's](https://github.com/storyblok/field-plugin/issues/392) description.

Reply:
Hmm 🤔 I actually don't know about external asset. Do you know @Dawntraoz maybe? I want to test a situation where is_external_url is true.

Reply:
Indeed @eunjae-lee, you need to go to the image field definition in the Block and check the Allow External URL, then when creating the story, instead of selecting an asset from the Assets Library, you should click on the `externalURL` icon and add a link to an image (for example for this one: https://letsenhance.io/static/8f5e523ee6b2479e26ecc91b9c25261e/1015f/MainAfter.jpg

Reply:
Thank you @Dawntraoz! I've followed your instruction, and I was able to upload an external asset.

![Screenshot 2024-09-17 at 14 07 35](https://github.com/user-attachments/assets/4d802aa4-65e9-41ac-87d7-ba8578663b42)

But, in asset selector, I don't see that. I feel like this should be normal, right? (It's just attached to the story as an external URL, but not really uploaded to my space).

So, if I understand this correctly, the selected asset can never be external. What do you think?

Reply:
You have a point since the message sent only involves Asset Manager iteration in the field plugin. It will always be `false` 🤔

Reply:
I guess that's why is_external_url even doesn't exist on storyfront right before it's passed down to field plugin.